src/plugins/filter/filter.style.scss

```
@@ -17,11 +20,16 @@ revogr-filter-panel {
   box-shadow: 0 5px 18px -2px var(--revo-grid-filter-panel-shadow, rgba(0, 0, 0, 0.15));
   padding: 10px;
   border-radius: 8px;
+  margin: 0;
 
   min-width: 220px;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)